src/components/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import '../styles/main.css';
+import AddTodo from './addtodo';
+import ListTodo from './listtodo';
+
+function App() {
+
+  const [todos, setTodos] = useState({
+    todoList: [],
+  });
+
+  const addNewTodo = (todo) => {
+    if (todo.title.length !== 0) {
+      setTodos((prev) => ({
+        todoList: [...prev.todoList, todo],
+      }));
+    }
+  };
+
+  const setCheck = (todo) => {
+    setTodos((prev) => ({
+      todoList: prev.todoList.map((item) => {
+        if (item.id === todo.id) {
+          return { ...item, completed: !item.completed };
+        }
+        return item;
+      }),
+    }));
+  };
+
+
+  return (
+    <div className="App d-flex">
+      <div className='container'>
+        <h1 className="todo-header">todos</h1>
+        <AddTodo addNewTodo={addNewTodo} todos={todos} />
+      <ListTodo todos={todos} setTodos={setTodos} changeHandler={setCheck}/>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] You didn't implement Routing in your project and this was in the project requirement ([Routing with React Router](https://ibaslogic.com/routing-with-react-router/)) , kindly Add Routing in your project

Reply:
Hello @RWUBAKWANAYO 

I really appreciate the feedback you gave me :pray: 
I am going to implement them

Cheers :ok_hand: